frontend/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopgate/terms-and-conditions",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
This is still 1.1.1. Please roll it back to 1.1.0.

Reply:
ah shoot. updated